gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NexusPageHeaderExample from './NexusPageHeaderExample';
+
+const nexusPageHeaderExampleCode = require('!!raw-loader!./NexusPageHeaderExample').default;
+
+const NxNexusPageHeaderPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxNexusPageHeader</code> is a React component encapsulating the Sonatype Nexus
+        branded page header logo and structure.
+      </p>
+      <h3 className="nx-h3 nx-tile__subsection-header">Props</h3>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Prop</th>
+            <th className="nx-cell nx-cell--header">Type</th>
+            <th className="nx-cell nx-cell--header">Required</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">productInfo</td>
+            <td className="nx-cell">object</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              An optional object containing at least one field: <code className="nx-code">name</code>, the product
```

Comment:
Is it really optional?

Reply:
> Is it really optional?

Probably not: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/96b2b2be0323fa7840f525fef295a3f1284c4ba7

Reply:
> Probably not: 96b2b2b

It still acts like the `version` field is optional.  I assume that isn't true either.

Reply:
> > Probably not: [96b2b2b](https://github.com/sonatype/sonatype-react-shared-components/commit/96b2b2be0323fa7840f525fef295a3f1284c4ba7)
> 
> It still acts like the `version` field is optional. I assume that isn't true either.

I think the version field should be optional

Reply:
> I think the version field should be optional

we don't have a mockup where it isn't present :shrug:

Reply:
> > I think the version field should be optional
> 
> we don't have a mockup where it isn't present 🤷

There actually is now